frontend/src/bundles/studio/components/player-controllers/player-controllers.tsx

```diff
@@ -0,0 +1,100 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
I think 'player-controls' is a better name, as 'controller' is more commonly associated with backend route logic.

Reply:
Yeah, you are right